src/controllers/auth-middleware.ts

```diff
@@ -0,0 +1,36 @@
+import { IHttpServerComponent } from "@well-known-components/interfaces";
+import { Context, StatusCode } from "../types";
+
+async function withAuthTokenValidation(
+  context: IHttpServerComponent.DefaultContext<Context<string>>,
+  next: () => Promise<IHttpServerComponent.IResponse>
+): Promise<IHttpServerComponent.IResponse> {
+  // Validate Authorization header
+  const authHeader = context.request.headers.get("Authorization");
```

Comment:
If we're hardcoding the bearer token in one of our static UIs, we must take into consideration that those UIs are packaged and deployed to NPM before being uploaded to the CDN, carrying the token with them.

Reply:
auth middleware removed entirely!